### DISCUSSION TITLE
Hotkeys for screen recording dont work

### DISCUSSION THREAD

**Discussion by mary-ng5519** [Genymobile/feedback#36] [Genymotion Desktop] [answered]
Dear team, please help fix the hotkey bugs for screen recording features
Version 3.3.1 | Revision 20221102-b65230d63
Tons of thanks
p/s: please help move this ticket to the right place/responsible people

**Comment by agavignet** [comment]
Hi @phathuynh5519 

Thanks for your bug report, we've decided to add this fix in the next patch release which should be out by the end of next week.

Best regards,
Adrien

**Comment by agavignet** [comment, marked as answer]
@phathuynh5519 For you information, this is fixed in Genymotion Desktop 3.3.2 (released today).
